src/annotator/guest.ts

```diff
@@ -921,4 +930,13 @@ export class Guest extends TinyEmitter implements Annotator, Destroyable {
       this.setHighlightsVisible(!this._highlightsVisible);
     }
   }
+
+  _showOutsideAssignmentNotice(show: boolean) {
```

Comment:
Two comments.

1) I guess this method should be marked as private.

```suggestion
  private _showOutsideAssignmentNotice(show: boolean) {
```

2) `showOutsideAssignmentNotice` sounds a bit like it's always going to be displayed, while in this case it is hidden if `show` is `false`. Maybe something like `setOutsideAssignmentNoticeVisibility` would be more clear?

Reply:
Yeah. Probably `toggleOutsideAssignmentNotice` would be more succinct in this context, and we have some other methods that follow this pattern.